README.md

```diff
@@ -13,3 +13,211 @@ This is an NWB extension for storing metadata of devices used in optical experim
+        <<OpticalFilter>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        center_wavelength_in_nm : numeric
+        bandwidth_in_nm : numeric
+    }
+    class EdgeOpticalFilter{
+        <<OpticalFilter>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        cut_wavelength_in_nm : numeric
+        slope_in_percent_cut_wavelength : numeric, optional
+        slope_starting_transmission_in_percent : numeric, optional
+        slope_ending_transmission_in_percent : numeric, optional
+    }
+    DichroicMirror *-- DeviceModel : extends
+    OpticalFilter *-- DeviceModel : extends
+    BandOpticalFilter *-- OpticalFilter : extends
+    EdgeOpticalFilter *-- OpticalFilter : extends
+```
+#### Devices in the microscopy setup
+```mermaid
+%%{init: {'theme': 'base', 'themeVariables': {'primaryColor': '#ffffff', "primaryBorderColor': '#144E73', 'lineColor': '#D96F32'}}}%%
+classDiagram
+    direction BT
+    class DeviceModel{
+        <<Device>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        model : text, optional
+    }
+    class ExcitationSource{
+        <<DeviceModel>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        illumination_type : text, optional
+        excitation_wavelength_in_nm : numeric, optional
+        power_in_W : numeric, optional
+        intensity_in_W_per_m2 : numeric, optional
+        exposure_time_in_s : numeric, optional
+    }
+    class PulsedExcitationSource{
+        <<DeviceModel>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        peak_power_in_W : numeric, optional
+        peak_pulse_energy_in_J : numeric, optional
+        pulse_rate_in_Hz : numeric, optional
+    }
+    class Photodetector{
+        <<DeviceModel>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        detector_type : text, optional
+        detected_wavelength_in_nm : numeric, optional
+        gain : numeric, optional
+        gain_unit : text, false
+    }
+    class ObjectiveLens{
+        <<DeviceModel>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        numerical_aperture : numeric, optional
+        magnification : numeric, optional
+    }
+    
+    ExcitationSource *-- DeviceModel : extends
+    PulsedExcitationSource *-- ExcitationSource : extends
+    Photodetector *-- DeviceModel : extends
+    ObjectiveLens *-- DeviceModel : extends
+    OpticalFilter *-- DeviceModel : extends
+    BandOpticalFilter *-- OpticalFilter : extends
+    EdgeOpticalFilter *-- OpticalFilter : extends
+    DichroicMirror *-- DeviceModel : extends
```

Comment:
Did you intend to duplicate the OpticalFilter and below in both the ophys and fiber photometry?

I think just including it in the separate block above is sufficient unless you also showed how those filters interact as metadata with the other extensions data series (which I think would be better in the respective extensions)

Reply:
Yes! Agree I can simplify it like this